reduce.js

```diff
@@ -0,0 +1,257 @@
+// # Exercises
+
+// 1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+
+// ```
+// Input: [1,2,3,4,5]
+// Output: [11,12,13,14,15]
+// ```
+
+const increasedTen = (arr) => {
+     return arr.reduce((newArray, el,i) => {
+      newArray[i]=(el + 10)
+      return newArray
+    }, [])
+   }
+   console.log(increasedTen([1,2,3,4,5]))
+
+// 2. Given an array, return a new array where every non-string element is converted to an empty string.
+
+// ```
+// Input: ["a", 123, "b", "c", {name: "cat"}]
+// Output: ["a", "", "b", "c", ""]
+// ```
+
+const noString = (arr) => {
+    return arr.reduce((newArray, el,i) => {
+     if (typeof el !== "string") {
+       newArray[i] = ""
```

Comment:
It's better to use the push method. newArray[i] would leave an empty index  for the elements that aren't strings.